src/doc/markdown.lisp

```diff
@@ -226,10 +226,13 @@
                  :regex "[<>&]")))
 
       (format stream "Methods:~%")
-      (loop :for (name . type) :in methods :do
-        (format stream "- <code>~A :: ~A</code>~%"
+      (loop :for (name . type) :in methods
+            :for docstring     :in method-docstrings :do
+
+        (format stream "- <code>~A :: ~A</code>~@[<br/>~A~]~%"
                 (html-entities:encode-entities (symbol-name name))
-                (to-markdown type))))
+                (to-markdown type)
+                docstring)))
```

Comment:
do we need to encode the do string? What happens if the docstring contains HTML?